### ISSUE TITLE
Output of summary.cross2 messed up in RStudio

### ISSUE THREAD

**Issue by kbroman** [rqtl/qtl2geno#142]
Consider this code:

```r
library(qtl)
sug <- read.cross("csv", "http://rqtl.org", "sug.csv", genotypes=c("CC", "CB", "BB"))
library(qtl2geno)
sug2 <- convert2cross2(sug)
summary(sug2)
```

In R from the command line, I get:

```
Object of class cross2 (crosstype "f2")

Total individuals            163
No. genotyped individuals    163
No. phenotyped individuals   163
No. with both geno & pheno   163

No. phenotypes                 5
No. covariates                 1
No. phenotype covariates       0

No. chromosomes               19
Total markers                 93
No. markers by chr:
 1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19
 5  7  5  5  5  4  8  4  4  5  6  3  3  5  5  4  4  6  5
```

But within RStudio I get:

```
Object of class cross2 (crosstype "f2")
Total individuals            163No. genotyped individuals    163No. phenotyped individuals   163No. with both geno & pheno   163

No. phenotypes                 5No. covariates                 1No. phenotype covariates       0

No. chromosomes               19Total markers                 93
No. markers by chr:
 1  2  3  4  5  6  7  8  9 10 11 12 13 14 15 16 17 18 19 
 5  7  5  5  5  4  8  4  4  5  6  3  3  5  5  4  4  6  5 
```

**Comment by kbroman** [comment]
Man, I'm doing some screwy shit in that function: `\b` to get text to align. 🙄

**Comment by kbroman** [comment]
Fixed in PR #143.
